coding/file_writer.hpp

```diff
@@ -31,8 +31,8 @@ class FileWriter : public Writer
                       Op operation = OP_WRITE_TRUNCATE, bool bTruncOnClose = false);
   ~FileWriter();
 
-  void Seek(int64_t pos);
-  int64_t Pos() const;
+  void Seek(uint64_t pos);
```

Comment:
Does this method override Writer::Seek()? If so, mark it as override. Same for other methods including dtor.